contrib/test/integration/build/cri-o.yml

```diff
@@ -77,3 +83,17 @@
     regexp: 'overlay\.override_kernel_check=1'
     state: present
   when: ansible_distribution == 'RedHat' or ansible_distribution == 'CentOS'
+
+- name: DUMMY TASK TO WORKAROUND CACHING OF AMI IN CI JOB
+  package:
+    name: "libselinux-python, policycoreutils-python"
+    state: latest
+
+- name: Enforce specific SELinux types for crio files on this platform
```

Comment:
Shouldn't this be handled by an RPM install?

Reply:
Unless I'm missing something, this is for testing from the compiled cri-o code.  So we need it both ways (in the RPM and in the playbook).  Though you're correct, if we're going to run tests after installing the RPM, this step isn't necessary.

Reply:
Ah, I understand